### PR TITLE
Added activities section

### LINKED ISSUE #363: Adding latest activities to contributors' section
**Actual Behaviour**

There's only a link to GitHub insights of the repo.

**Expected Behaviour**

Adding latest activities in GitHub which related to this repo in the contributors' section.

**Would you like to work on the issue?**

Yes.

### PR DESCRIPTION
**Pull request by STEdirisuriya**
- [x] Included a Preview link and screenshot showing after and before the changes.
- [x] Included a description of the change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- https://stedirisuriya.github.io/gci18.fossasia.org/
- Fixes #363 

## Description / Changes
- Added activities section with GitHub API.

### CODE REVIEW

**Comment by Ryan10145** [comment]
@STEdirisuriya Should the Thank You for Contributing section be moved below the new activities section? I feel like the Thank You for Contributing section fits into the footer and looks out of place now that is in the middle of the website.

**Comment by STEdirisuriya** [comment]
@Ryan10145 I have added it before the thank you section,
@fragm3 I have removed unwanted file changes. Could you please check it again, if you're free?
and also could you please tag this issue as an advance, https://github.com/fossasia/gci18.fossasia.org/issues/363

Thank you!

File: css/custom.css [hunk 1]
```diff
@@ -438,3 +438,44 @@ p.person-name {
...
+  margin-left: 74px;
+}
+.issues-wrapper .right-side {
+  float: right;
+  text-align: right;
+  margin-top: -55px;
+}
+.issues-wrapper i {
+  margin-right: 5px;
+}
```
Comment (Ryan10145) [inline css/custom.css]: Add a new line at the end of the file

File: js/scripts.js [hunk 2]
```diff
@@ -1437,3 +1437,46 @@ $(document).ready(function(){
...
+            html += "<p>Opened by </p>";
+            html += "<a href='"+pull.user.html_url+"' target='_blank' class='user'>";
+            html += pull.user.login+"</a><div class='right-side'>";
+            html += "<a href='"+pull.html_url+"' target='_blank'>";
+            html += "<i class='fa fa-eye'></i>See";
+            html += "</a></div></div>";
+            $(".issues-wrapper").append(html);
+        });
+    });
+});
```
Comment (Ryan10145) [inline js/scripts.js]: Add a new line at the end of the file

**Comment by STEdirisuriya** [comment]
@Ryan10145 I have resolved these issues, thank you for pointing me them.

**Comment by STEdirisuriya** [comment]
@Ryan10145 @fragm3 @nikit19 @jogendra Thank you all for reviewing my PR!

**Pull request merged**